### ISSUE TITLE
CHANGELOG

### ISSUE THREAD

**Issue by ATriantafyllopoulos** [autrainer/autrainer#30]
I think it's important to keep a CHANGELOG with all the major changes per version, especially now that the package is changing so fast. Do we already have one @ramppdev ?

**Comment by ramppdev** [comment]
> You mean this here: https://github.com/autrainer/autrainer/releases/tag/v0.3.0?
> 
> This is based on the commit message, right? So it just takes the one-liner we use as title?

Yeah exactly, i left it as the default autogenerated version which just uses the commit titles which in our case should always be PRs. Additionally we have https://github.com/autrainer/autrainer/compare/v0.2.0...v0.3.0.
But there is no detailed description of what changed without clicking through the PRs and reading the descriptions ...

**Comment by ATriantafyllopoulos** [comment]
Yeah, I think we should add a `CHANGELOG.rst`  and include it in the docs automatically, that makes it easier for people to work with the docs and get an overview of what has changed..
